README.md

```diff
@@ -23,16 +23,16 @@ using that port. Usually this happens if you have apache or nginx installed in y
     sudo service apache2 stop
     sudo service nginx stop
 
-If the Apache runs locally it binds the port and Docker will be blocked. If upu need to run the Apache at the same time there is a need to fork the dockert3kit repo and add another port to docker-compose file to expose web container.
+If the Apache runs locally it binds the port and Docker will be blocked. If you need to run the Apache at the same time there is a need to fork the dockert3kit repo and add another port to docker-compose file to expose web container.
 
 ## Starting Docker containers
 
 To get Docker container up and running:
 
     cd ~/path_to_your_project/shared/local.your-site-name
-    composer require --dev lauri/dockert3kit '~2.2.2'
+    composer require --dev lauri/dockert3kit '~3.0.1'
```

Comment:
Hello @laurisaarni, 

I think it is good for `lauri/dockert3kit '~3.0.1` we can put directory `composer.json` in package `t3kit_composer` , so that we do not need to to run composer install two time. What do you think about this?
cc: @MattiasNilsson, @manmath